.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Thank you for contributing to GRPC!
+Please provide the following information to help us make the most of your pull request:
```

Comment:
there should be a link to https://github.com/grpc/grpc/blob/master/CONTRIBUTING.md

Reply:
Yes. Thank for suggesting!